pkg/osquery/runtime/osqueryinstance.go

```diff
@@ -679,7 +679,7 @@ func calculateOsqueryPaths(rootDirectory string, runId string, opts osqueryOptio
 	// See: https://github.com/kolide/launcher/issues/1599
 	osqueryFilePaths := &osqueryFilePaths{
 		pidfilePath:           filepath.Join(rootDirectory, fmt.Sprintf("osquery-%s.pid", runId)),
-		databasePath:          filepath.Join(rootDirectory, "osquery.db"),
+		databasePath:          filepath.Join(rootDirectory, fmt.Sprintf("osquery-%s.db", runId)),
```

Comment:
TODO -- will move this to a separate PR, there is more work there that needs to be done